### PR TITLE
feat: merged feedback section into report issues and changed google form

### LINKED ISSUE #1214: Merge Feedback section into Report Issues and rename
**Actual Behaviour**
Report issues currently only shows form to report issue

**Expected Behaviour**
A dropdown for user to select to report issue or give feedback
Forms to report issue or give feedback should change accordingly
Report Issues should be renamed to "Feedback & Bugs"

**Would you like to work on the issue?**
@abhinavraj23

### PR DESCRIPTION
**Pull request by abhinavraj23**
Fixes #1214 

**Changes**:
Merged feeback section into report issues and changed google form

**Screenshot/s for the changes**: [Add screenshot/s of the layout where you made changes or a `*.gif` containing a demonstration]
 

 

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: [Compress the `app-debug.apk` file into a `<feature>.rar` or `<feature>.zip` file and upload it here]
[feedback_bugs.apk.zip](https://github.com/fossasia/pslab-android/files/2189845/feedback_bugs.apk.zip)

### CODE REVIEW

**Comment by abhinavraj23** [comment]
@CloudyPadmal @cweitat @harsh-2711 @Avjeet @Vikum94 Please review, I have also made changes in the Google form.

Here is the screenshot of feedback section

**Comment by cweitat** [comment]
I have checked the Google Form and i realise that the radio button is not a "Required" field. Resulting user able to go to the next section even if never select on any radio button. 
@CloudyPadmal radio button is fine.
But please take note that when I say drop-down and you have another suggestion please voice beforehand before doing.
@abhinavraj23 if you noticed, I referenced #1213 in #1214. Did you read #1213 ? It says to rename the section for Help & Feedback, not remove!

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -11,8 +11,7 @@
     <string name="nav_instruments">Instruments</string>
     <string name="nav_settings">Settings</string>
     <string name="nav_about_us">About Us</string>
-    <string name="nav_report">Report Issues</string>
-    <string name="nav_help_feedback">Help &amp; Feedback</string>
```
Comment (cweitat) [inline app/src/main/res/values/strings.xml]: Why remove?

Comment (abhinavraj23) [reply app/src/main/res/values/strings.xml]: Okay @cweitat I do the changes :+1:

Comment (abhinavraj23) [reply app/src/main/res/values/strings.xml]: @cweitat changes done :+1:  , I have renamed the section.

**Comment by abhinavraj23** [comment]
@cweitat I have also added the required field in the question part

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -35,7 +35,7 @@
         <item>Device</item>
         <item>Settings</item>
         <item>PSLab</item>
-        <item>Help &amp; Feedback</item>
```
Comment (cweitat) [inline app/src/main/res/values/strings.xml]: This part?

**Comment by harsh-2711** [comment]
@abhinavraj23

**Comment by harsh-2711** [comment]
@abhinavraj23 Can we use "Feedback and bug reporting" as a title in side navigation drawer

**Comment by abhinavraj23** [comment]
Yes @harsh-2711 nice idea, what do you think @cweitat @CloudyPadmal

**Comment by cweitat** [comment]
The text is too long. Anyway to shorten it? 
Or Feedback & Report Bugs?

**Comment by abhinavraj23** [comment]
Yes @cweitat I think feedback and report bugs is better, what do you think @harsh-2711 @CloudyPadmal

**Comment by harsh-2711** [comment]
Yes that sounds good :+1:

**Pull request merged**